### DISCUSSION TITLE
Unable to enter SMS verification code for authentication process.

### DISCUSSION THREAD

**Discussion by forrby** [homebridge-simplisafe3/homebridge-simplisafe3#368] [Q&A] [answered]
Hello,

I am completely new to all of this. Just trying to get Simplisafe to work with homekit. So I installed Homebridge on my Macbook, and downloaded the simplisafe3 plugin. I'm trying to start the authentication process. After entering my email, password and pressing "Start Authentication" I get stuck with a message that says "Attempting to login with credentials..." When I press "start authentication" I do get a SMS with a verification code from 
Simplisafe buy I have no idea where to enter this code. Any help would enourmously appreciated!! (And sorry if this is a super dumb question...)
![Screen Shot 2022-11-17 at 7 24 53 PM](https://user-images.githubusercontent.com/118493783/202595405-c8a0f22f-2a67-4607-8d49-254d7956abf5.png)
![Screen Shot 2022-11-17 at 7 20 29 PM](https://user-images.githubusercontent.com/118493783/202595407-992be85e-b141-40e3-a439-531afd32e85f.png)
![Screen Shot 2022-11-17 at 7 20 23 PM](https://user-images.githubusercontent.com/118493783/202595409-9139cc3b-50dd-4d2b-9df1-a922eae97fde.png)

**Comment by PoppyS33d** [comment]
Yep same issue here. I was able to downgrade to 1.10.8 and authenticate the old way but within a few hours the devices had all gotten “no response.” There seems to be a new version out .11 in the last few hours, and was able to authenticate just fine with it, but we’ll see if it stays online…

**Comment by shamoon** [comment, marked as answer]
Duplicate of #369, unfortunately SS have changed things, I’m not sure I’ll be able to get the “auto” login working again (ever) so we’re back to manual for now :/
